spec/core/Platform.Spec.savi

```diff
@@ -9,6 +9,14 @@
       if Platform.is_macos   (1 | 0) +
       if Platform.is_windows (1 | 0)
 
+  :it "detects FreeBSD, OpenBSD, NetBSD and Dragonfly"
+    if Platform.is_bsd (assert: (
+      Platform.is_freebsd ||
+      Platform.is_openbsd ||
+      Platform.is_netbsd  ||
+      Platform.is_dragonfly
+    ))
```

Comment:
I believe that of these four platforms are mutually exclusive, right?

If so, then I think a test like the following would be more robust, and cover more failure cases:
```suggestion
    is_bsd U8 = if Platform.is_bsd (1 | 0)
    assert: is_bsd == U8[0] +
      if Platform.is_freebsd   (1 | 0) +
      if Platform.is_openbsd   (1 | 0) +
      if Platform.is_netbsd    (1 | 0) +
      if Platform.is_dragonfly (1 | 0)
```

That is, this tests that *exactly one* of those specific BSD platforms is true *if and only if* the overall platform designation is BSD - and if it's not BSD, *then none of them are true*.